.github/workflows/build-and-push.yml

```
@@ -0,0 +1,42 @@
+name: "Build and push image"
+on:
+  push:
+    branches: master
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

It is probably a good idea to also do:
```suggestion
        uses: actions/checkout@v2
        with:
          persist-credentials: false
```
It defaults to persisting credentials; while it's harmless here (we only build on `master`), it's a good habit to get into when we don't need to push back to the repo.

---

ok

---

Done with commit aba4539d8492508904fee3d842002ce3bee7f8ec